### PR TITLE
Integration specs

### PR DESCRIPTION
**Pull request by Bantealem**
- [x] In this pull request i have done the following 
- [x] I have done integration test  for User index page
- [x] I have done integration test  for User show page
- [x] I have done integration test  for Post index page
- [x] I have done integration test  for Post  show page
-  
image

### CODE REVIEW

File: spec/features/post_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    @post = Post.create!(author: @user1, title: 'Web Development', text: 'Learn Web developmet at Microverse',
+                         comments_counter: 1, likes_counter: 0)
+    Comment.create!(author_id: @user1.id, post_id: @post.id, text: 'this is first comment')
+    visit user_posts_path(user_id: @user1.id, id: @post.id)
+  end
+  describe 'User index page' do
+    it 'I can see the user username.' do
+      expect(page).to have_content(@user1.name)
+    end
+
```
Comment (tufoinnkuo10) [inline spec/features/post_show_spec.rb:16]: - [ ] Great job so far. you are almost there.  for featured tests for User post show page, you are missing test for;
- Tests that asserts you can see who wrote the post.
- Tests that asserts you can see the username of each commentor
- Tests that asserts you can see the comment each commentor left.
please kindly add that.
nice work 👏🏾

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @Bantealem 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Spec files ok


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @Bantealem, 

Your project now is complete, good job implementing the required changes!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bantealem** [comment]
Thank you.

**Pull request merged**